development/mimir-ingest-storage/config/grafana-agent.flow

```diff
@@ -0,0 +1,107 @@
+prometheus.scrape "metrics_local_mimir_read_write_mode_mimir_write" {
```

Comment:
This file was generated by running 

```
docker run -v ./config:/config -e AGENT_MODE=flow grafana/agent:v0.40.0 convert -f static /config/grafana-agent.yaml
```

from `development/mimir-ingest-storage` directory, and then adding 

```
         enable_protobuf_negotiation = true
         scrape_classic_histograms = true
```

lines to all scrape configurations.